examples/multiple_windows/test/multiple_windows_test.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: invalid_use_of_internal_member
+
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+// ignore: avoid_relative_lib_imports
+import '../lib/main.dart' as multiple_windows;
+import 'package:flutter/src/foundation/_features.dart' show isWindowingEnabled;
+
+void main() {
+  if (!isWindowingEnabled) {
+    const String windowingDisabledErrorMessage = '''
+Windowing APIs are not enabled.
+
+Windowing APIs are currently experimental. Do not use windowing APIs in
+production applications or plugins published to pub.dev.
+
+To try experimental windowing APIs:
+1. Switch to Flutter's main release channel.
+2. Turn on the windowing feature flag.
+
+See: https://github.com/flutter/flutter/issues/30701.
+''';
+    // ignore: avoid_print
+    print(windowingDisabledErrorMessage);
+    exit(0);
+  }
+
+  testWidgets('Multiple windows smoke test', (WidgetTester tester) async {
+    multiple_windows
+        .main(); // builds the app and schedules a frame but doesn't trigger one
```

Comment:
The formatter's formatting is a bit funky with long trailing comments, consider moving the comment above (here and all subsequent tests)

```suggestion
    // Builds the app and schedules a frame but doesn't trigger one
    multiple_windows.main();
```